ledger-core/virtual/execute/execute.go

```diff
@@ -357,13 +336,31 @@ func (s *SMExecute) stepDeduplicate(ctx smachine.ExecutionContext) smachine.Stat
 	}
 
 	if isDuplicate {
-		return ctx.Error(throw.E("duplicate found as pending request"))
```

Comment:
Why?

Reply:
because duplicate is not an error, and later on duplicate check is not an error too